### DISCUSSION TITLE
[question]  sum up  rows?

### DISCUSSION THREAD

**Discussion by georgew21** [nocodb/nocodb#3592] [Q&A]
I am searching web and docs of nocodb and I can't find how I can sum up rows. 

For example I have this:
date ---  amount
1/3/2021 --- 25 euros
2/3/2021 ---- 30 euros

How can I add a row at the end aggregating the numbers ?

thanks!

**Comment by o1lab** [comment]
You can use [rollup column](https://docs.nocodb.com/setup-and-usages/rollup)
